app/views/comments/_single_comment.html.erb

```diff
@@ -14,9 +14,9 @@
     <% elsif !can_see_hidden_comment?(single_comment) %>
       <p class="message"><%= ts("(This comment is under review by an admin and is currently unavailable.)") %></p>
     <% else %>
-      <% cache([single_comment, single_comment.comment_owner], expires_in: 1.week) do %>
-        <% # FRONT END, update this if a.user comes in %>
-        <h4 class="heading byline">
+      <%# FRONT END, update this if a.user comes in %>
```

Comment:
Is there any value in keeping around this 12 years old comment?

Reply:
I don't even know what it means so I decided to leave it 🤷‍♂️